.github/workflows/build.yaml

```diff
@@ -72,6 +72,11 @@ jobs:
   build:
     runs-on: ubuntu-latest
     steps:
+      # See https://github.com/NASA-IMPACT/hls-base/blob/24b9251d1a09c111eeb14836   1376d3705b391f28/.github/workflows/build_push_to_ecr.yml
+      # See https://github.com/actions/runner-images/issues/2840#issuecomment-790492173
+      - id: pre_setup
+        name: Create more disk space
+        run: sudo rm -rf /usr/share/dotnet && sudo rm -rf /opt/ghc && sudo rm -rf "/usr/local/share/boost" && sudo rm -rf "$AGENT_TOOLSDIRECTORY"
       - id: setup
         name: Setup
         uses: scientist-softserv/actions/setup-env@v0.0.17
```

Comment:
Are we gonna cut a new release? If so then these will need to point at the cut-version, we are currently clocked at @v0.0.19, so @v0.0.20?

Reply:
I'm planning on doing that